torchtune/modules/rlhf/utils/_convert_weights.py

```diff
@@ -76,17 +81,17 @@ def _permute(t, n_heads):
     return converted_state_dict
 
 
-def mistral_reward_tune_to_hf(
+def reward_tune_to_hf(
```

Comment:
I wonder if there's a better location for these - our other converters are in models/convert_weights. PEFT adapter conversions were added there. I think it makes sense to also throw these in there, or do you think that will make the file bloated?

Reply:
Yeah I was thinking similar. I think it does make the file a bit bloating and would reflect the sentiment from my comment about; we could revisit this and put weight conversion somewhere more general when we need it.